src/components/routes/RoutesMapToolbar.vue

```diff
@@ -0,0 +1,209 @@
+<script lang="ts">
+/**
+ * RoutesMapToolbar Component
+ *
+ * @description * Use this component to render a toolbar for a RoutesMap.
+ *
+ * @props
+ * - `drawEnabled` (boolean, required): Enabled state for draw tool.
+ * - `deleteEnabled` (boolean, required): Enabled state for delete tool.
+ *
+ * @events
+ * - `update:draw-enabled`: Emitted when draw tool is toggled.
+ * - `update:delete-enabled`: Emitted when delete tool is toggled.
+ * - `undo`: Emitted when undo button is clicked.
+ * - `save`: Emitted when save button is clicked.
+ * - `current-position`: Emitted when current position button is clicked.
+ *
+ * @example
+ * <routes-map-toolbar />
+ */
+
+// libraries
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'RoutesMapToolbar',
+  emits: [
+    'update:draw-enabled',
+    'update:delete-enabled',
+    'undo',
+    'save:route',
+    'current-position',
+  ],
+  props: {
+    drawEnabled: {
+      type: Boolean,
+      default: false,
+    },
+    deleteEnabled: {
+      type: Boolean,
+      default: false,
+    },
+  },
+});
+</script>
+
+<template>
+  <div>
+    <!-- Toolbar: Top -->
+    <div
+      class="flex justify-center absolute-top q-pa-sm"
+      :style="{ zIndex: 1, pointerEvents: 'none' }"
+      data-cy="toolbar-top"
+    >
+      <q-toolbar
+        class="col-auto gap-8 q-pa-sm"
+        :style="{
+          borderRadius: '9999px',
+          backgroundColor: 'white',
+          pointerEvents: 'auto',
+        }"
+      >
+        <!-- Button: Enable draw (draw route) -->
+        <q-btn
+          dense
+          round
+          unelevated
+          class="q-pa-none q-ma-none"
+          color="transparent"
+          text-color="primary"
+          @click.prevent="$emit('update:draw-enabled', !drawEnabled)"
+          data-cy="add-route-button"
+        >
+          <q-avatar
+            size="32px"
```

Comment:
`size` attribute value 32px reapet, you can const for this attr

Reply:
Using const.